File: app/src/main/java/com/silvertak/relationshipsmanager/customInterface/ManagingGroupListener.java
package com.silvertak.relationshipsmanager.customInterface;

import com.silvertak.relationshipsmanager.data.RelationshipGroupInfo;

public interface ManagingGroupListener {
    void onInfoClick(RelationshipGroupInfo info);
    void onDeleteClick(RelationshipGroupInfo info);
    void onModifyClick(RelationshipGroupInfo info);
}

File: app/src/main/java/com/silvertak/relationshipsmanager/customInterface/ManagingGroupListener.kt
package com.silvertak.relationshipsmanager.customInterface

import com.silvertak.relationshipsmanager.vo.RelationshipGroupInfo

interface ManagingGroupListener {
    fun onInfoClick(info: RelationshipGroupInfo?)
    fun onDeleteClick(info: RelationshipGroupInfo?)
    fun onModifyClick(info: RelationshipGroupInfo?)
}